Framework/AODMerger/src/aodMerger.h

```diff
@@ -10,14 +10,18 @@
 // or submit itself to any jurisdiction.
 
 #include <TString.h>
+#include <TRegexp.h>
 
 const char* removeVersionSuffix(const char* treeName)
 {
   // remove version suffix, e.g. O2v0_001 becomes O2v0
+  // handle O2track_iu correctly
```

Comment:
So track_iu becomes track. And wasn't this an unintended but needed feature?

E.g. AmbiguousTracks has a column TrackId which in the AO2D needs to have the offset re-indexed (in https://github.com/AliceO2Group/AliceO2/blob/024bf20093dbe75d501a1670f60eeb8699645b80/Framework/AODMerger/src/aodMerger.cxx#L377) of track_iu. However, the column is called fTrackId which would not be found in the new version anymore.

Can you try to run the merged with this change? It should complain about unbound indices...

Reply:
You are right. I will drop this change and with it the sanity-check at the end.